getting-started/Installation/configuration.md

```diff
@@ -67,6 +67,6 @@ secrets:
 
 With Amber you can encrypt your environment setting `amber encrypt {envrionment}`, this command will open your editor to allow make changes if needed and then encrypt the file `{envrionment}.enc`.
 
-A `{project_name}/.encryption_key` file is provided containing a secret key base this is by default added to your `.gitignore` so it does not get commited to your repository. Without the encryption key, you won't be able to decrypt your environment settings.
+A `{project_name}/.encryption_key` file is provided containing a secret key base this is by default added to your `.gitignore` so it does not get committed to your repository. Without the encryption key, you won't be able to decrypt your environment settings.
```

Comment:
A `{project_name}/.encryption_key` file is provided. It contains a `secret_key_base` that is used to decrypt your encrypted environment settings. This file is added to `.gitignore` so it will not be committed to your repository.
Without the encryption key, you won't be able to decrypt your environment settings.